addons/explosives/CfgAmmo.hpp

```diff
@@ -27,10 +27,13 @@ class CfgAmmo {
     class PipeBombCore: TimeBombCore;
     class PipeBombBase: PipeBombCore;
     */
+    // GVAR(size) = 0; is small size
+    // GVAR(size) = 1; is large size
```

Comment:
This should be put into the [framework page of documentation](https://ace3mod.com/wiki/framework/explosives-framework.html).